List.h

```diff
@@ -0,0 +1,80 @@
+//
+//  List.h
+//  List
+//
+//  Created by Conor Sweeney on 4/21/20.
+//  Copyright © 2020 csweeney. All rights reserved.
+//
+
+#include <string>
+#include <deque>
+#include <vector>
+#include <map>
```

Comment:
Use an unordered_map instead of map since lookups and inserts are O(log(n)) for map compared to O(1) for unordered_map.